deploy/remove_saf.sh

```diff
@@ -1,25 +1,30 @@
 #!/bin/sh
 #
-# Removes SAF and amq7 cert manager from your cluster
+# Removes SAF and (optionally) amq7 certmanager from your cluster
 #
-SAF_PROJECT=${SAF_PROJECT:-sa-telemetry}
+OCP_PROJECT=${OCP_PROJECT:-sa-telemetry}
+REMOVE_CERTMANAGER=${REMOVE_CERTMANAGER:-true}
 
-# The whole SAF project
-oc delete project "${SAF_PROJECT}"
+# The whole SAF project (start this first since it's slow)
+oc delete project "${OCP_PROJECT}"
```

Comment:
I was trying to figure out how to create a `Namespace` object so that we didn't specifically need to rely on the `oc` commands. Never could get it to work though.

Reply:
I played with that and concluded that an openshift project is more than just a namespace and I wasn't going to rip it apart anymore than that.